packages/service/src/index.ts

```diff
@@ -1,59 +1,63 @@
 import * as YAML from 'js-yaml'
 import * as fs from 'fs'
-import API from '@mesg/api';
-import { IApi } from '@mesg/api/lib/types';
-import * as bs58 from '@mesg/api/lib/util/base58';
+import * as grpc from 'grpc'
+import * as protoLoader from '@grpc/proto-loader'
```

Comment:
code generation doesn't work?

Reply:
I tried with protobufjs but got a few issues when trying to send through the RPC implementation (grpc). Proto loader + grpc works fine and the types are not really necessary here as they are not exposed to the end-user. It could definitely be added later but didn't want to make this PR bigger for something with such a low impact.